bigbluebutton-tests/playwright/customparameters/customparameters.js

```diff
@@ -234,6 +234,79 @@ class CustomParameters extends MultiUsers {
     expect(await this.modPage.getLocator(e.selectCameraQualityId).inputValue()).toBe('low');
     await this.modPage.waitAndClick(e.startSharingWebcam);
   }
+
+  async breakoutRooms() {
+    await this.modPage.waitAndClick(e.manageUsers);
+    await this.modPage.wasRemoved(e.createBreakoutRooms);
+  }
+
+  async liveTranscription() {
+    const { liveTranscription } = getSettings();
+    test.fail(!liveTranscription, 'Live Transcription is disabled');
```

Comment:
Tests working pretty well! I'd say only to remove this check because even with the live transcription setting defined as `false` the test should pass since this parameter will force the same behavior

```suggestion
```